siad/update.go

```diff
@@ -31,50 +31,50 @@ func newerVersion(version string) bool {
 		// inputs are trusted, so no need to check the error
```

Comment:
wrt TODO: make this more robust

We could always pick a convention like updates always follow 3 nums: x.y.z, and then we explicitly ignore everything after the third bullet. If there's something important enough to push an update through, it's worth increasing the 'z'